src/pages/CompanyFinancial.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getFinancial } from '../redux/CompanyFinancialSlice';
+import Company from '../components/singlecompany';
+import Header from './Navbar';
+
+const CompanyFinancial = () => {
+  const { financialstats } = useSelector((state) => state.finance);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getFinancial());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Header />
+      {financialstats.map((comp) => (
+        <Company
+          key={comp.id}
+          id={comp.id}
+          companySymbol={comp.companySymbol}
+          reportedCurrency={comp.reportedCurrency}
+          calendarYear={comp.calendarYear}
+          netIncome={comp.netIncome}
+        />
+      ))}
+    </>
+  );
```

Comment:
- [ ] as the previous reviewer asked you should add an search bar or filter feature that we can filter the data with it as it is one of the requirements in order to complete the requirements 
- Screenshot of the requirements 
- [ ][ optional ] as one of the requirements is to follow the design provided you should follow colors, font-family, font-size and composition, and space between elements.
- Screenshot of the Requirements 
<img width="497" alt="Screenshot 2023-03-23 at 13 27 56" src="https://user-images.githubusercontent.com/61790587/227219696-42f5232e-7471-4ddc-bdcc-d3316ee068b4.png">

- Font provided with colors 
<img width="1137" alt="Screenshot 2023-03-23 at 13 30 15" src="https://user-images.githubusercontent.com/61790587/227219743-0d12c6eb-b9d5-4451-8de5-ef49e9fac958.png">

<img width="498" alt="Screenshot 2023-03-23 at 12 00 32" src="https://user-images.githubusercontent.com/61790587/227200028-c1ab6779-e746-4065-a35d-c328d6734df8.png">


- [ ] The Details  page should be the same as the one in the design by styling the background, font, colors , the data elements 
- yours 
<img width="398" alt="Screenshot 2023-03-23 at 13 30 37" src="https://user-images.githubusercontent.com/61790587/227219896-22a0b6d6-e5c1-46cb-872c-1cdd024451f1.png">

- Design 
<img width="207" alt="Screenshot 2023-03-23 at 13 29 56" src="https://user-images.githubusercontent.com/61790587/227219940-7274e0fb-cffd-466a-ae54-9481bcc48eba.png">

Reply:
The search bar is added and working. thank you.